README.rst

```diff
@@ -616,6 +616,18 @@ candidate list further down.
         pkg:golang/github.com/gorilla/context@234fd47e07d1004f0aed9c#api
 
 
+- `hackage` for Haskell packages
+
+  - The default repository is `https://hackage.haskell.org`.
+  - The `version` is package version.
+  - The `name` is case sensitive and use kebab-case
```

Comment:
Is there a doc that tells something about case sensitivity? Also can you explain  what is  kebab-case?

Reply:
Yeah sure, like snake case use '_' kebab case use '-', more info can be found here

Reply:
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Reply:
I don't have any doc, but I tried to find a package name by changing it's letter cases, it does not identified it, 'AC-HalfInteger' & 'ac-halfInteger' were not same, so I concluded it from here